### ISSUE TITLE
How to use options in "tool" mode?

### ISSUE THREAD

**Issue by frou** [liquidz/antq#104]
I've tried both:
`clojure -Tantq outdated :upgrade true`
and:
`clojure -Tantq outdated '{:upgrade true}'`

They don't seem to have any effect that's different to:
`clojure -Tantq outdated`

There's no interactive prompt and no writes to the `deps.edn` file.

**Comment by liquidz** [comment]
@frou Thanks for your reportng!
Oh, it's buggy. I'll fix it soon.

**Comment by liquidz** [comment]
@frou I've fixed this problem, and cut a new release #105 

Could you try as follows?
```sh
clojure -Ttools install com.github.liquidz/antq '{:git/tag "0.16.3"}' :as antq
clojure -Tantq outdated :upgrade true
```

**Comment by frou** [comment]
That now works for me. Thank you, I appreciate it!
